ts/client/src/scripts/mb-admin.ts

```diff
@@ -509,6 +509,45 @@ async function makePerpMarketReduceOnly() {
   );
 }
 
+async function makePerpMarketUntrusted() {
+  const result = await buildAdminClient();
+  const client = result[0];
+  const admin = result[1];
+  const creator = result[2];
+
+  const group = await client.getGroupForCreator(creator.publicKey, GROUP_NUM);
+  const perpMarket = group.getPerpMarketByName('BTC-PERP');
+  await client.perpEditMarket(
+    group,
+    perpMarket.perpMarketIndex,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    false,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
+    null,
```

Comment:
This is hard to review. I know it's not a priority, but this horrible API really sticks out to me every time I see it. Might be worthwhile to spend some time to change this to something where unchanged parameters don't need to be mentioned.

Separate PR though.